index.html

```diff
@@ -33,5 +42,139 @@ <h3>I'm a software Developer, </h3>
+        <img class="yoga-image" src="./final-images/Img Placeholder.png" alt="a lady doing yoga">
+        <h2>Multi-Post Stories</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+          has been the industry's standard dummy text ever since the 1500s, when an unknown printer took
+          a standard dummy text.
+        </p>
+        <ul class="stack-holder">
+          <li><a href="#">CSS</a></li>
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+        <button class="see-project"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-four">
+      <div class="about-me-ctn">
+        <h1>About me</h1>
+        <p>
```

Comment:
- [ ] Excellent job on applying the proper font styles to the title and paragraph! However, there is still some adjustment needed in terms of spacing:

    - The space between the title and the paragraph is currently larger than what is specified in the design. Please reduce the space to match the design guidelines more closely.

![image](https://github.com/prince-rutagengwa/mobile-portfolio/assets/55840999/0478f6e3-2a81-468f-ae09-4175e262e989)

Remember to pay attention to the details, and keep up the great work! 👏🌟